src/test/splitting/splitting/find_task_start_indices_test.py

```diff
@@ -1,11 +1,11 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
-from typing import List
-
+import pytest
 import pandas as pd
```

Comment:
Pytest and pandas are not the standard libraries, so please, move these imports into a separated second group